### ISSUE TITLE
code block in quote creates false positives for MD052

### ISSUE THREAD

**Issue by davidhewitt** [rvben/rumdl#114]
Consider the following markdown snippet:

``````markdown
```rust
#[derive(Debug)]
struct Ok;
```

> ```rust
> #[derive(Debug)]
> struct NotOk;
> ```
``````

The code block inside the block quote (on `NotOk`) creates a false positive for link reference:

```
[MD052] Reference 'derive(Debug)' not found
```

**Comment by rvben** [comment]
Hi @davidhewitt,

Good catch - code blocks inside blockquotes were being incorrectly parsed for reference links. This was fixed in 0.0.162.

Thanks for reporting!
